.gitignore

```diff
@@ -55,4 +55,4 @@ services/**/dist
 app.yaml
 gcp/data-service/**/dist
 gcp/data-service/**/lib
-gcp/data-service/tsconfig.build.tsbuildinfo
\ No newline at end of file
+gcp/data-service/tsconfig.build.tsbuildinfo
```

Comment:
remove `incremental: true` from the tsconfig and then you can delete the tsbuildinfo